cityofaustin/atd-moped
# Use project list view query hooks for csv export

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/14068

This PR ports the existing CSV export generation into a custom hook. Lots of the existing logic was reused but moved out of the `Search` component.

## Testing
**URL to test:** 

Local

**Steps to test:**
1. Spin up the stack locally [with production data](https://app.gitbook.com/o/-LzDQOVGhTudbKRDGpUA/s/-MIQvl_rKnZ_-wHRdp4J/dev-guides/how-tos/how-to-load-production-data-into-a-local-instance). Seed data can be used but there will be fewer projects to test this feature on. You can also compare prod exports to the export that come out of this branch if you use prod data.
2. Try exporting data with different order by columns
3. Try exporting data with simple search filters applied
4. Try exporting data with advanced search filters applied
5. You should see the number of records in the export match the total number of results returned and filtered down to your searches.

---
#### Ship list
